src/systemtest/resources/noDateRange/expected/reposense_testrepo-Charlie_master/commits.json

```diff
@@ -8697,10 +8481,10 @@
     }
   },
   "authorContributionVariance": {
-    "charlesgoh": 86818.71,
-    "jeffreygohkw": 71305.99,
-    "Esilocke": 413377.22,
-    "wangyiming1019": 63797.973
+    "charlesgoh": 87747.24,
+    "jeffreygohkw": 72073.97,
+    "Esilocke": 419051.7,
+    "wangyiming1019": 64918.33
```

Comment:
Why is the variance changing though?

Reply:
@yamidark 
I have investigated it.
I think it is float operation

Consider this arguments
`--repo https://github.com/reposense/RepoSense.git --since 01/03/2019`
The variance for the old implementation will be `2149.7754`
The variance for the new implementation will be `2149.7756`

I think the change is caused by the different order of evaluation of `totalContribution`
For the old implementation, the order will be as follow:
`0 0 0 0 0 0 0 0 174 0 0 0 0`
While for the new implementation, the order will be as follow:
`0 174 0 0 0 0 0 0 0 0 0 0 0`

Reply:
Fixed, I change implementation